dlt/destinations/impl/bigquery/bigquery.py

```diff
@@ -49,7 +46,7 @@ class BigQueryTypeMapper(TypeMapper):
         "timestamp": "TIMESTAMP",
         "bigint": "INTEGER",
         "binary": "BYTES",
-        "wei": "BIGNUMERIC",  # non parametrized should hold wei values
```

Comment:
I don't think we need to change american english to british english or vice versa in the docs ;)